charts/workflows/charts/move2kube/templates/01-sonataflow_m2k.yaml

```diff
@@ -255,11 +255,15 @@ spec:
   podTemplate:
     container:
       resources: {}
-      image: quay.io/orchestrator/serverless-workflow-move2kube:4fae4bd85a48f4091f5c67a83fbcc94dcd29f1f2
+      image: {{ .Values.workflow.image }}
```

Comment:
same as above

Reply:
On second thought, no, we should not remove it

At least to be consistent with what is done in the orchestrator charts: https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/values.yaml#L157

@masayag WDYT?

Reply:
The greeting workflow is going to be removed from the orchestrator chart. image can be included, overall it is generated from a specific git-hash. otherwise, we'll have to update that information on chart's level.

Reply:
it is there ATM since the images of sonataflow weren't stable for some time.
we can remove it from there now.
---
not saying it isn't a good practice, but since we're dealing with 2 repos, there will need to be some changes to support it.